prisma-fmt/tests/code_actions/scenarios/multi_schema_one_model_one_enum/result.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
Do we miss the enum fix here?

Reply:
Yep! this was due to enum qf not filtering the correct diagnostic per [above](https://github.com/prisma/prisma-engines/pull/3525#discussion_r1054631400)